### DISCUSSION TITLE
Does Immich support IPTC keywords?

### DISCUSSION THREAD

**Discussion by GD-Dal** [immich-app/immich#5021] [General]
Does Immich support IPTC keywords?
I use Lightroom to add keywords to my pictures, and as I understand it, they are stored in IPTC.
If it is supported, I haven't found a way to show them yet.

Thanks

**Comment by bverkron** [comment]
Did you ever find a solution? I'm in the same boat with files from lightroom. I can't get any keywords to show up in Immich. In fact I can't determine if / where Immich stores keywords _at all._

**Reply by GD-Dal** [reply]
Hi.
This was resolved back in version 0.113 I think. It works quite well, but unfortunately not on shared albums yet.
But you have to turn this feature on before it shows up, by going to Account Settings -> Features -> Tags
